docker/Dockerfile-debug

```diff
@@ -2,19 +2,19 @@ FROM fiware/orion-ld-base
 
 ARG PATH_TO_SRC='opt/orion/'
 
-
 RUN cp -a /usr/local/lib/. /usr/lib/x86_64-linux-gnu/
 RUN rm -r /usr/local/lib/
 
-ADD ./docker/debug/gdbinit gdbinit
-RUN yum -y --nogpgcheck install gdb
-
 COPY . ${PATH_TO_SRC}
 
 WORKDIR ${PATH_TO_SRC}
 
 RUN make debug install
 
-ENTRYPOINT ["gdb", "-x", "gdbinit", "--args", "orionld", "-fg", "-multiservice", "-ngsiv1Autocast"]
+RUN mkdir -p /var/{log,run}/orionld
```

Comment:
why do you need /var/log?

Reply:
I probably don't (the broker doesn't use it).
Copied that line from Dockerfile-ubi

Reply:
I assumed some library might use it

Reply:
Hopefully not:)

Reply:
Fixed in e0ae19c